ext-apiserver/pkg/apis/cluster/v1alpha1/cluster_types_test.go

```diff
@@ -15,65 +14,68 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
 package v1alpha1_test
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	. "k8s.io/kube-deploy/ext-apiserver/pkg/apis/cluster/v1alpha1"
 	. "k8s.io/kube-deploy/ext-apiserver/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
```

Comment:
Can you change this dot import too?

https://github.com/golang/go/wiki/CodeReviewComments#import-dot